README.md

```diff
@@ -41,7 +41,7 @@ steps:
     fail_ci_if_error: true # optional (default = false)
     verbose: true # optional (default = false)
   env:
-    token: ${{ secrets.CODECOV_TOKEN }}
+    CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
@thomasrockhu-codecov why is it an env var and not the token input as described in the arguments section below?

Reply:
See https://github.com/codecov/codecov-action/pull/1241